Sources/Public/CalendarView.swift

```diff
@@ -429,7 +451,10 @@ public final class CalendarView: UIView {
   }
 
   private var isReadyForLayout: Bool {
-    bounds.width > 0 && bounds.height > 0
+    // There's no reason to attempt layout unless we have a non-zero `bounds.size`. We'll have a
+    // non-zero size once the `frame` is set to something non-zero, either manually or via the
+    // Auto Layout engine.
+    bounds.size != .zero
```

Comment:
Changing this since we need `layoutSubviews` to get far enough so that we hit this line and API consumers can learn about the max month height https://github.com/airbnb/HorizonCalendar/pull/150/files#diff-733b4844a228b4fda6b27e4f88b42a6fc4b04eb589081da658741416a2152bd9R242

If API consumers implement their placeholder height constraint like this `calendarView.heightAnchor.constraint(equalToConstant: 0)`, without this change, `isReadyForLayout` will be `false` and we won't make it far enough in `layoutSubviews` for us to calculate a `maximumMonthHeight` to pass back to API consumers, leaving the `CalendarView` stuck rendering nothing.

I think as long as we have a non-zero `bounds.size`, we have enough info to start computing parts of the layout, which will also allow API consumers to get that `maximumMonthHeightDidChange` even if they used 0 height as their placeholder.